plugins/techdocs-react/src/hooks.ts

```diff
@@ -55,12 +55,12 @@ export const useShadowRootElements = <
       });
     }
     return () => observer?.disconnect();
-  }, [shadowRoot, render, rerender]);
+  }, [shadowRoot]);
 
-  if (!shadowRoot) return [];
+  if (!root || !(root instanceof HTMLElement)) return [];
 
   return selectors
-    .map(selector => shadowRoot.querySelectorAll<TReturnedElement>(selector))
+    .map(selector => root.querySelectorAll<TReturnedElement>(selector))
```

Comment:
since `root` now refers to the shadow root's first child, wouldn't this break usages where you're querying for elements that sit in the shadow root's second and beyond children?

Reply:
In reality the structure for the techdocs is `<html><body>...</body></html>` so the first child should be the HTML element